lib/assets/javascripts/twine.coffee

```diff
@@ -110,12 +110,9 @@
 
     callbacks = currentBindingCallbacks
 
-    # IE and Safari don't support node.children for DocumentFragment and SVGElement nodes.
-    # If the element supports children we continue to traverse the children, otherwise
-    # we stop traversing that subtree.
-    # https://developer.mozilla.org/en-US/docs/Web/API/ParentNode.children
-    # As a result, Twine are unsupported within DocumentFragment and SVGElement nodes.
-    bind(context, childNode, if newContextKey? then null else indexes) for childNode in (node.children || [])
+    # IE and Safari don't support node.children for DocumentFragment or SVGElement,
+    # See explanation in childrenForNode()
+    bind(context, childNode, if newContextKey? then null else indexes) for childNode in childrenForNode(node)
```

Comment:
I guess we could compare the before/after list and warn about it in dev mode?  Doesn't seem worth it to me, but I don't have to troubleshoot Twine issues 😛

Reply:
I think it should be OK, we'd still have to tophat everything to see those messages